lxd/instance_backup.go

```diff
@@ -134,6 +134,10 @@ func instanceBackupsGet(d *Daemon, r *http.Request) response.Response {
 		return resp
 	}
 
+	if d.cluster.LocalNodeIsEvacuated() {
```

Comment:
In general I don't think we need all of those changes. If there are instances left on an evacuated server, I don't mind having the ability to still interact with them in theory.

What we need to prevent however is the creation of any new instances, so that part should stay.